tools/m2kcli/commands/analog/analog_in.cpp

```diff
@@ -0,0 +1,423 @@
+					  "\n"
+					  "Positional arguments:\n"
+					  "  uri                   describe the context location \n"
+					  "                        auto | ip:192.168.2.1 | usb:XX.XX.X\n"
+					  "Optional arguments:\n"
+					  "  -h, --help            show this help message and exit\n"
+					  "  -q, --quiet           return result only\n"
+					  "  -v, --voltage channel=<index>... raw=<value>\n"
+					  "                        print the voltage of the given channel\n"
+					  "                        channel - {0 | 1}\n"
+					  "                        raw - 0 (processed values)\n"
+					  "                            - 1 (raw values)\n"
+					  "  -c, --capture channel=<index>... buffer_size=<size> raw=<value> [format=<type>]\n"
+					  "                        print a specific number of samples\n"
+					  "                        format - {csv | binary}; default csv\n"
+					  "  -g, --get [<attribute>...]\n"
+					  "                        return the value of the specified global attributes\n"
+					  "                        attribute:\n"
+					  "                            sampling_frequency\n"
+					  "                            oversampling_ratio\n"
+					  "                            trigger_source\n"
+					  "                            trigger_delay\n"
+					  "                            all\n"
+					  "  -G, --get-channel channel=<index> [<attribute> ...]\n"
+					  "                        return the value of the attributes corresponding to the given channel\n"
+					  "                        attributes:\n"
+					  "                            range\n"
+					  "                            trigger_level\n"
+					  "                            trigger_analog_condition\n"
+					  "                            trigger_mode\n"
+					  "                            trigger_hysteresis\n"
+					  "                            all\n"
+					  "  -s, --set [<attribute>=<value>...]\n"
+					  "                        set the value of the specified global attributes\n"
+					  "                        attribute:\n"
+					  "                            sampling_frequency - {10 | 100 | ... | 100000000}\n"
+					  "                            oversampling_ratio - int\n"
+					  "                            trigger_source - {channel_1 | channel_2 | channel_1_or_channel_2 | channel_1_and_channel_2 | channel_1_xor_channel_2}\n"
+					  "                            trigger_delay - int\n"
+					  "                            kernel_buffers - int\n"
+					  "  -S, --set-channel channel=<index> [<attribute>=<value> ...]\n"
+					  "                        set the value of the specified attributes corresponding to the given channel\n"
+					  "                        attributes: \n"
+					  "                            range - {high | low}"
+					  "                            trigger_level - int"
+					  "                            trigger_analog_condition - {raising_edge | falling_edge | low_level | high_level}\n"
```

Comment:
typo, I think? Should be "rising_edge". Using "raising_edge" in the m2kcli causes the tool to crash.

Reply:
Also.. some missing "\n". Not functional.

Reply:
The help message was corrected in commit a61547e.